deploy/docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM alpine:3.5
+
+LABEL maintainer="adundovi@gmail.com"
+LABEL description="Prepares a docker image \
+of pump.io server, http://pump.io/."
+
+ENV LC_ALL en_US.UTF-8
+
+RUN apk add --no-cache \
+  nodejs \
+  git \
+  python \
+  make \
+  g++ \
+  graphicsmagick
+RUN npm install -g databank-mongodb
+RUN npm install -g pump.io
```

Comment:
I'm unfamiliar with node.js waters, therefore I don't know when and how often do you push something to npm, but I generally avoid the source tree in something which is intended for "just work" environment because it could be unstable or even undeployable while one wouldn't push something in that state in a package manager, right? Or you suggest pulling the source from a tagged release?